service-api/src/main/java/greencity/client/RestClient.java

```diff
@@ -327,8 +328,21 @@ public void save(UserVO userVO, String accessToken) {
      */
     private HttpHeaders setHeader() {
         String accessToken = httpServletRequest.getHeader(AUTHORIZATION);
+        Cookie[] cookies = httpServletRequest.getCookies();
+        String uri = httpServletRequest.getRequestURI();
+        if (cookies != null && uri.startsWith("/management")) {
+            accessToken = getTokenFromCookies(cookies);
+        }
         HttpHeaders headers = new HttpHeaders();
         headers.set(AUTHORIZATION, accessToken);
         return headers;
     }
+
+    private String getTokenFromCookies(Cookie[] cookies) {
+        String token = Arrays.stream(cookies)
+            .filter(c -> c.getName().equals("accessToken"))
+            .findFirst()
+            .map(Cookie::getValue).orElse(null);
+        return token == null ? null : "Bearer " + token;
+    }
 }
```

Comment:
It is best to create another findNotDeactivatedByEmail method with the accessToken parameter in the AccessTokenAuthenticationFilter class.